### DISCUSSION TITLE
Access subreddits within multireddits

### DISCUSSION THREAD

**Discussion by NomarCub** [Docile-Alligator/Infinity-For-Reddit#898] [Ideas]
I can see and edit what subreddits a multireddit contains, but I can't click on them or access them in any other way.
Somewhat related to https://github.com/Docile-Alligator/Infinity-For-Reddit/issues/293.

**Comment by sjain882** [comment]
+1 from me - this would be brilliant to have, it's sad this got buried.

I have 300+ subscriptions on Reddit, so grouped them into multireddits.

Unfortunately, multireddits are far from perfect.

They don't just show you most of the top posts from the day from each sub; if one sub has a lot more activity than the other, it will push a lot of those posts to you, even if most are basic text posts with 3-4 upvotes only. Other subreddits often get buried under all of these and you never see them.

Implementing this would double the value of multireddits, in that they can be used as category lists instead. This is the only thing stopping me from using the app as my main reddit app!

Being able to long press on a multireddit, or even tap a button on the side of each multireddit in the multireddit list, and get a full list of subs in there would be great, rather than having to go into Edit Multireddit.

I believe there is a strong use case for this, would really appreciate it if the developer could implement this.
